src/components/overlayCookie/overlayCookie.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import { translate } from 'react-i18next';
+import 'i18n/i18n';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faTimes } from '@fortawesome/free-solid-svg-icons';
+import styles from './overlayCookie.scss';
+
+class OverlayCookie extends Component {
+  state = {
```

Comment:
do you guys want to enable class properties in the linter? this is valid with that syntax

Reply:
Yes @charlesjliu (I imagine, not 100% sure what you're talking about because TBH I've never configured a linter). Babel and ES6 can deal with this syntax now and it's less boilerplate :).
https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599

I can change it for now if you want to make this PR pass right away but pls let's not start building the whole app with unnecessary constructors.

Reply:
I'll leave it for @crysfel to decide! But I think this would be the preferred syntax for most (what you've been using)

Reply:
My current PR updates the linter, however after that there's another failure in Travis only, I need to investigate what the issue is.

I'd prefer not to overwrite the constructor, in most cases is not necessary and I always recommend avoiding boilerplate code. Please leave it like this, it looks good!